features/opennms-es-rest/main-module/src/main/resources/OSGI-INF/blueprint/mainModuleBlueprint.xml

```diff
@@ -29,6 +29,8 @@
       <cm:property name="archiveAlarmChangeEvents" value="true" />
       <cm:property name="archiveOldAlarmValues" value="true" />
       <cm:property name="archiveNewAlarmValues" value="true" />
+      <cm:property name="timeout" value="5000" /> <!-- 5 second timeout for Elasticsearch operations -->
```

Comment:
What do you think about renaming this to `retryTimeout` instead? Since it doesn't actually affect the timeout of the client operations, i.e. the connections or the reads.